docs/reference/module-api/module-overview.md

```diff
@@ -603,6 +626,8 @@ module.exports = {
 };
 ```
 
+If including a route option, use an object. See the [route options](#route-options) section for more.
```

Comment:
Should include a guide to the error names that can be thrown, if I'm not missing it somewhere, which is very possible.

Reply:
I'll note to do this in a separate PR. I'm not sure how we would want to handle that since it applies to other scenarios (rignt?).